<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>891</width>
     <height>841</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 229, 229);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 229, 0);
background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>641</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>7</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <row>
    <property name="text">
     <string>id</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Ответственный</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Тип работы</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Тема работы</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Элемент</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Время начала работы</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>153</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Исходные данные</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>258</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Заполните финальные данные</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>631</width>
     <height>121</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <row>
    <property name="text">
     <string>Время завершения</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Ссылка на файл</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Комментарий</string>
    </property>
   </row>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <zorder>textBrowser_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>tableWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
